trace_decoder/src/iface.rs

```diff
@@ -0,0 +1,185 @@
+//! Public types for this crate.
```

Comment:
What's `iface`? `interface`? I don't think 4 more characters would hurt if that's the case :)